### PR TITLE
[GETUNAME] Update Romanian translation (2nd attempt)

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
This is the second attempt to split the file in chunks.

### CODE REVIEW

**Comment by HBelusca** [comment]
Comment from @binarymaster:

> My general comment for this PR would be... why there are more added lines than removed?
> Is there something you added from yourself? I'm not sure this is how translations should be done

**Comment by AndreiMiloiu** [comment]
The file was incomplete. More lines that are available in the English file were not available in the Romanian one, so I had to add them before making the translation of the file. The English, French, Spanish, Hungarian files (and maybe other files, too) are complete, but not the Romanian one.

File: dll/win32/getuname/lang/ro-RO.rc [hunk 1]
```diff
@@ -7574,8 +7574,8 @@ BEGIN
     8692 "Săgeată spre dreapta cu cerc mic"
     8693 "Săgeată în jos lângă săgeată în sus"
     8694 "Trei săgeți spre dreapta"
-    8695 "Săgeată spre stânga cu tăietură verticală"
-    8696 "Săgeată spre dreapta cu tăietură verticală"
+    8695 "Săgeată spre stânga tăiată verticală"
+    8696 "Săgeată spre dreapta tăiată verticală"
```
Comment (reactos-contributor) [inline dll/win32/getuname/lang/ro-RO.rc:7578]: To me, this change looks a bit hard to understand, especially when you preferred „cu tăietură” before (see resource № 2098 & 7661). Please take care of consistency.

File: dll/win32/getuname/lang/ro-RO.rc [hunk 2]
```diff
@@ -7983,6826 +7983,9449 @@ BEGIN
...
+    9946 "Semnul de conducere lentă"
+    9947 "Triunghi alb îngroșat în jos"
+    9948 "Intrarea din stânga închisă"
+    9949 "Saltire pătrată"
+    9950 "Diagonală care se încadrează în cerc alb în pătrat negru"
+    9951 "Camion negru"
+    9952 "Intrare din stânga restricționată 1"
+    9953 "Intrare din stânga restricționată 2"
+    9954 "Simbolul astronomic pentru Uranus"
+    9955 "Cerc îngroșat cu contur și două puncte deasupra"
```
Comment (reactos-contributor) [inline dll/win32/getuname/lang/ro-RO.rc:8837]: I think you want to reconsider „contur” as translation for „stroke”. In many other instances you've used „tăiat” or „cu tăietură”.

File: dll/win32/getuname/lang/ro-RO.rc [hunk 3]
```diff
@@ -7983,6826 +7983,9449 @@ BEGIN
...
-    42852 "Latin Capital Letter Thorn With Stroke"
-    42853 "Latin Small Letter Thorn With Stroke"
-    42854 "Latin Capital Letter Thorn With Stroke Through Descender"
-    42855 "Latin Small Letter Thorn With Stroke Through Descender"
-    42856 "Latin Capital Letter Vend"
-    42857 "Latin Small Letter Vend"
-    42858 "Latin Capital Letter Et"
-    42859 "Latin Small Letter Et"
-    42860 "Latin Capital Letter Is"
-    42861
```
Comment (reactos-contributor) [inline dll/win32/getuname/lang/ro-RO.rc:9625]: Here you start using „ruptură” as another translation for „with stroke”. Before you've used „tăiat”/„cu tăietură”. Please take care of consistency.

Comment (reactos-contributor) [inline dll/win32/getuname/lang/ro-RO.rc:9631]: This must be a typo. You most likely meant „curbată” (as „curvată” means something else).

Comment (reactos-contributor) [inline dll/win32/getuname/lang/ro-RO.rc:9633]: I am all for translating as much as possible, but I am not sure „Tiny” and „Miny” (as mathematical operators used in combinatorial game theory) should. See [Tiny and miny on Wikipedia](https://en.wikipedia.org/wiki/Tiny_and_miny)

**Review by reactos-contributor — CHANGES_REQUESTED** [review]
I made some comments, on what I reviewed so far. I haven't finished, so I'll probably come back later with more.

File: dll/win32/getuname/lang/ro-RO.rc [hunk 4]
```diff
@@ -1410,8 +1410,8 @@ BEGIN
     1415 "Ligatură armenească eci yiwn"
     1417 "Punct armenesc"
     1418 "Cratimă armenească"
-    1421 "Right-Facing Armenian Eternity Sign"
-    1422 "Left-Facing Armenian Eternity Sign"
+    1421 "dreapta-Facing Armenian Eternity Sign"
+    1422 "stânga-Facing Armenian Eternity Sign"
```
[the remark this answers is not in the document]
Comment (AndreiMiloiu) [reply dll/win32/getuname/lang/ro-RO.rc:1414]: Sorry, I will correct it. Thank you for telling me this.

**Comment by AndreiMiloiu** [comment]
I hope I solved all the mistakes you mentioned until now. I uploaded more translated words, so look at the last version.

**Comment by AndreiMiloiu** [comment]
Sorry for commit [Lines 55246-64217](https://github.com/reactos/reactos/pull/5313/commits/9b0c8d4c290a94836b74ca2064ef304141eaee49).
I mistakenly made a too big chunk. However, if I remember well everything, it's only my contribution. Everything was in English before. I promise that the next chunks will be smaller than this one.

**Comment by reactos-contributor** [comment]
> _"I promise that the next chunks will be smaller than this one."_

Well, as of now, the ability to see your work on the file (and add any comments) is not available here until you reduce it to whatever limit GitHub has.

**Comment by AndreiMiloiu** [comment]
So, do I have to create a new PR again?

**Comment by reactos-contributor** [comment]
> So, do I have to create a new PR, again? I just wanted to make your work easier, not harder.

Not necessarily. The simplest solution for you would be to have two branches. The current branch to have this file with changes limited to resources up to around what I last commented on, and the newly created branch to contain the rest. Then you go on working carefully on each version separately (only review fixes on this one, active translation on the other).
Do not forget update this PR afterwards, so that review could continue.

**Comment by AndreiMiloiu** [comment]
I don't know how to do two branches. I corrected some mistakes I discovered afterwards in my file and I made some changes according to your suggestion. I will create a new PR.

**Comment by reactos-contributor** [comment]
> I don't know how to do two branches.

Yes, tooling may be challenging at times (and especially in the beginning) to understand and use. Use whatever you can, for now.
Whenever you'll have enough patience for getting acquainted with branches, [here is how GitHub explains how to work with them](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository).

**Comment by AndreiMiloiu** [comment]
Thank you for the information. I will have a look to the link you sent me.

**Pull request closed without merging**